### ISSUE TITLE
STRETCH: Delete Functionality

### ISSUE THREAD

**Issue by GonzalesMatthew** [GonzalesMatthew/pet-adoption#4]
- Add a DELETE button to each of your cards
- When a user clicks the button, remove the card from the DOM and from the Data Array

**Comment by GonzalesMatthew** [comment]
Add a DELETE button to each of your cards
When a user clicks the button, remove the card from the DOM and from the Data Array

**Comment by GonzalesMatthew** [comment]
Actually looking at this closer it works, but not well.

- Consider that the delete functionality, when displaying filtered pets, appears to splice the wrong pet from the array!
- Consider that you would want a filtered view to remain filtered after you delete a pet (splice from array)
   - the current logic does not refilter so must consider a logical way for this to happen

TLDR: filtered view may splice incorrect pet from the array. filtered views should ideally remain filtered.

Reopening this
